components/ResultsPageComponents/StaffSuggestions/StaffSuggestionsChart.jsx

```diff
@@ -0,0 +1,258 @@
+      bubbleTop: "135px",
+      bubbleLeft: "648px",
+      bubbleBackgroundColour: "#366F99",
+      bubbleDiameter: "95px",
+      bubbleTextColour: "#FFFFFF",
+      textMarginLeft: "15%",
+      textFontSize: "12px",
+      textString: "Improving bike/walk infrastructure",
+      valueMarginLeft: "25%",
+      valueFontSize: "18px",
+      valueString: "12%",
+    },
+    {
+      key: 6,
+      bubbleTop: "230px",
+      bubbleLeft: "613px",
+      bubbleBackgroundColour: "#9BB7CC",
+      bubbleDiameter: "109px",
+      bubbleTextColour: "#FFFFFF",
+      textMarginLeft: "15%",
+      textFontSize: "12px",
+      textString: "Arrangements for family members",
+      valueMarginLeft: "25%",
+      valueFontSize: "18px",
+      valueString: "8%",
+    },
+    {
+      key: 7,
+      bubbleTop: "308px",
+      bubbleLeft: "485px",
+      bubbleBackgroundColour: "#022640",
+      bubbleDiameter: "177px",
+      bubbleTextColour: "#FFFFFF",
+      textMarginLeft: "15%",
+      textFontSize: "24px",
+      textString: "Other",
+      valueMarginLeft: "25%",
+      valueFontSize: "48px",
+      valueString: "13%",
+    },
+  ];
+
+  const quotes = [
+    {
+      key: 1,
+      radiusSE: true,
+      quoteTop: "10px",
+      quoteLeft: "105px",
+      quote:
+        "Better end of trip facilities.  A locker that I can store suit and shirts, and a damp towel to dry in.",
+    },
+    {
+      key: 2,
+      radiusSE: true,
+      quoteTop: "130px",
+      quoteLeft: "5px",
+      quote:
+        "Incentivize car pooling - complete a stat dec and get a discount on your rates.",
+    },
+    {
+      key: 3,
+      radiusNE: true,
+      quoteTop: "480px",
+      quoteLeft: "60px",
+      quote: "More direct public transport options to the workplace.",
+    },
+    {
+      key: 4,
+      radiusSW: true,
+      quoteTop: "10px",
+      quoteLeft: "725px",
+      quote: "I am 20 mins from the station so car pool could work well.",
+    },
+    {
+      key: 5,
+      radiusSW: true,
+      quoteTop: "100px",
+      quoteLeft: "740px",
+      quote: "Provide dedicated bike lanes on arterial roads.",
+    },
+    {
+      key: 6,
+      radiusSW: true,
+      quoteTop: "210px",
+      quoteLeft: "725px",
+      quote:
+        "I have 2 toddlers in tow that need to be dropped off at childcare I can't think of any option other than driving.",
+    },
+    {
+      key: "7a",
+      radiusNW: true,
+      quoteTop: "400px",
+      quoteLeft: "663px",
+      quote:
+        "Flexibility to work from home for three days,  maybe one week a month.",
+    },
+    {
+      key: "7b",
+      radiusNW: true,
+      quoteTop: "485px",
+      quoteLeft: "580px",
+      quote: "Move to electric powered vehicles.",
+    },
+  ];
+
+  function QuoteBox(props) {
+    const {
+      radiusNW,
+      radiusNE,
+      radiusSE,
+      radiusSW,
+      quoteTop,
+      quoteLeft,
+      quote,
+    } = props;
+    const cornerCurve = "30px";
+    return (
+      <Box
```

Comment:
If we replace `Box` with `Flex` inside the bubbles and bubble text, does it get more responsive? 

Happy to approve it in the current state as it's already a great improvement (well done @RCode-Blue !) if @hqtan is happy with everything.

Reply:
Doesn't make a difference.

The bubbles are positioned _absolute_ relative to the parent's _relative_, which was the only way I could arrange them the way they are. The upside is we can make different arrangements or show/hide bubbles or quotes depending on the breakpoint.

![image](https://user-images.githubusercontent.com/33863416/186039327-2db96c39-c5ca-406d-bd16-77f07fcbb977.png)

Reply:
Good idea, maybe we can remove the text on mobile version and use `click` or `hover` eventListener to show it, not for this version, for the next iteration.